### ISSUE TITLE
Input Password shows password when hits enter.

### ISSUE THREAD

**Issue by tjapa** [dohomi/tamagui-kitchen-sink#18]
I dont know if it's a bug in my code or it's the default behavior of the component, but when I fill the input password and hit enter, the password show/hide the password.

**Comment by dohomi** [comment]
I don't see your bug inside the demo: https://tamagui-extras.vercel.app/?path=/story/form-input--password 
can you show your code?

**Comment by tjapa** [comment]
I tried a codesandbox but I'm having some trouble to run the app: https://codesandbox.io/p/sandbox/admiring-framework-cuw0t4?file=%2Fpackages%2Fapp%2Ffeatures%2Fhome%2Fscreen.tsx&selection=%5B%7B%22endColumn%22%3A20%2C%22endLineNumber%22%3A28%2C%22startColumn%22%3A20%2C%22startLineNumber%22%3A28%7D%5D

Here's a sample of the code:
```tsx
          <Form
            space
            onSubmit={() => {
              console.log('tedst')
            }}
          >
            <LmInput />

            <LmInput />
            <LmInput isPassword />

            <Form.Trigger asChild>
              <Button>Test</Button>
            </Form.Trigger>
          </Form>

```

A video that shows the behavior:

https://user-images.githubusercontent.com/17579637/231906445-88ef05ca-a2f9-4dac-97de-098a98cc6d68.mp4

**Comment by dohomi** [comment]
did you follow this example: https://github.com/dohomi/tamagui-kitchen-sink/blob/master/packages/ui/stories/components/forms/LmRhf.stories.tsx#L168

I will try to see if I can reproduce it within that story

**Comment by dohomi** [comment]
fixed in 0.14.1
